### PR TITLE
Theatre 76 repertoire page

### PR DESCRIPTION
**Pull request by Kate-Kryvopustova**

### CODE REVIEW

File: src/app/components/repertoire/repertoire.component.scss [hunk 1]
```diff
@@ -1,53 +1,77 @@
...
 
-.container {
-  padding: 0;
+.wrapped__active-nav {
+  margin: 10px;
+  font-family: 'Montserrat', serif;
+  font-size: 40px;
+  font-weight: 600;
+  font-stretch: normal;
+  font-style: normal;
```
Comment (Serhio77777) [inline src/app/components/repertoire/repertoire.component.scss:22]: mixins

File: src/app/components/repertoire/repertoire.component.scss [hunk 2]
```diff
@@ -1,53 +1,77 @@
...
-  height: 16.429rem;
-  width: 11.643rem;
-  margin-bottom: 6.214rem;
+.perfomance__item {
   cursor: pointer;
+}
+
+.item__wraper {
+  width: 350px;
+  height: 500px;
```
Comment (Serhio77777) [inline src/app/components/repertoire/repertoire.component.scss:42]: mixins

File: src/app/components/repertoire/repertoire.component.ts [hunk 3]
```diff
@@ -13,6 +13,7 @@ import { LoaderService } from '../partials/spinner/loader.service';
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class RepertoireComponent implements OnInit {
+  p = 1;
```
Comment (Serhio77777) [inline src/app/components/repertoire/repertoire.component.ts:16]: What is p?

File: src/app/components/repertoire/repertoire-nav-header/repertoire-nav-header.component.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
+<div class="nav-header-container">
```
Comment (Serhio77777) [inline src/app/components/repertoire/repertoire-nav-header/repertoire-nav-header.component.html:1]: Is that component the same as the Header at the festivals?

**Review by Serhio77777 — CHANGES_REQUESTED** [review]

File: src/app/components/repertoire/repertoire-nav-header/repertoire-nav-header.component.scss [hunk 5]
```diff
@@ -0,0 +1,28 @@
+.nav-header-container {
+  display: flex;
+  justify-content: center;
+  background-color: #b49864;
+  height: 56px;
+}
+
+.item {
+  padding: 0 15px;
+  font-family: 'Montserrat';
```
Comment (yana-tsylinska) [inline src/app/components/repertoire/repertoire-nav-header/repertoire-nav-header.component.scss:10]: variables

**Review by yana-tsylinska — CHANGES_REQUESTED** [review]

File: src/app/components/repertoire/repertoire.component.scss (hunk 2 above)
Comment (Kate-Kryvopustova) [reply src/app/components/repertoire/repertoire.component.scss]: Done

File: src/app/components/repertoire/repertoire.component.scss (hunk 1 above)
Comment (Kate-Kryvopustova) [reply src/app/components/repertoire/repertoire.component.scss]: Done

File: src/app/components/repertoire/repertoire.component.ts (hunk 3 above)
Comment (Kate-Kryvopustova) [reply src/app/components/repertoire/repertoire.component.ts]: Done

File: src/app/components/repertoire/repertoire-nav-header/repertoire-nav-header.component.scss (hunk 5 above)
Comment (Kate-Kryvopustova) [reply src/app/components/repertoire/repertoire-nav-header/repertoire-nav-header.component.scss]: Done

**Review by yana-tsylinska — APPROVED** [review]

**Pull request merged**